ui/editor/src/ctrl.ts

```diff
@@ -56,14 +56,17 @@ export default class EditorCtrl {
     this.initialFen = (cfg.fen || params.get('fen') || INITIAL_FEN).replace(/_/g, ' ');
 
     this.redraw = () => {};
+    if (!this.cfg.embed) {
+      this.options.orientation = (params.get('color') === 'black') ? 'black' : 'white'
```

Comment:
Don't do this if the board editor is embedded like in the `Create Chapter` dialog